skill-tree /src/main/java/com/RDS/skilltree/Endorsement/EndorsementController.java

```diff
@@ -20,8 +22,13 @@
 public class EndorsementController {
     private final EndorsementService endorsementService;
 
+    @GetMapping(value = "/")
+    public List<EndorsementModel> getEndorsements() {
+        return endorsementService.getEndorsements();
```

Comment:
- Please convert this DTO. 
- Should we have same response structure? i.e., GenericResponse< List<EndorsementDTO>> ?

Reply:
This will be done in the next upcoming PRs for this feature after the DTO is finalized